config/jobs/kubernetes/sig-cluster-lifecycle/packages.yaml

```diff
@@ -15,3 +15,19 @@ periodics:
       - ./tests/e2e/packages/verify_packages_published.sh
       image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
       imagePullPolicy: Always
+
+- name: periodic-kubernetes-e2e-packages-install-deb
+  interval: 24h
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  spec:
+    containers:
+    - args:
+      - --timeout=60
+      - --repo=k8s.io/kubeadm=master
+      - --scenario=execute
```

Comment:
This one should be especially easy to switch to the [pod utilities](https://github.com/kubernetes/test-infra/blob/master/prow/pod-utilities.md#how-to-configure).

Instead of scenario execute:
- set `decorate: true` on the prowjob (EG below `interval`)
- drop all of these flags
- set this instead of args on the pod:
```yaml
command:
- ./tests/e2e/packages/verify_packages_install_deb.sh
```
- checkout the repo with the following (on the prowjob, not the pod)
```yaml
extra_refs:
 - org: kubernetes
   repo: kubeadm
   base_ref: master
   # might not need this, but for reference this will set the checkout on disk to the alias
   path_alias: k8s.io/kubeadm
```
- drop the unnecessary presets

IE:
```yaml
- name: periodic-kubernetes-e2e-packages-install-deb
  interval: 24h
  decorate: true
  spec:
    containers:
    -  image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
       imagePullPolicy: Always
       command:
       - ./tests/e2e/packages/verify_packages_install_deb.sh
```

Reply:
thanks for the info. updated.
made this job and the existing package job use this method.